### PR TITLE
add: home page and changed images

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
Hello,

I have created a landing page for my e-commerce app. 
Please review and suggest.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -113,3 +107,29 @@ main {
 .text-decor-none{
     text-decoration: none;
 }
+
+.card-img-horizontal {
+    min-width: 13.5rem;
+    max-width: 13.5rem;
+}
```
Comment (anik31) [inline styles.css]: Why have you used same value for both min-width & max-width ?

File: index.html [hunk 2]
```diff
@@ -64,5 +64,214 @@
...
+          <div class="container fade">
+            <img class="carousel-img" src="/images/carousel-3.jpg" />
+          </div>
+
+          <div class="container">
+            <img class="carousel-img" src="/images/carousel-4.jpg" />
+          </div>
+
+          <div class="container">
+            <img class="carousel-img" src="/images/carousel-5.jpg" />
```
Comment (anik31) [inline index.html:84]: add alt to images

Comment (ApurvaSawant11) [reply index.html:84]: Done. Added alt attribute to images

File: styles.css (hunk 1 above)
Comment (ApurvaSawant11) [reply styles.css]: Fixed the width for image

**Review by anik31 — APPROVED** [review]
Good work @ApurvaSawant11 . UI is very nice & code is also good.

File: css/header.css [hunk 3]
```diff
@@ -1,6 +1,6 @@
 .sub-header {
     background: var(--primary);
-    color: var(--white);
+    color: var(--white-color);
     padding: 0.3rem;
```
Comment (b0llu) [inline css/header.css]: if the value is less than 1rem you can use px like 3px

File: css/home.css [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+.left,
+.right {
+  cursor: pointer;
+  position: absolute;
+  top: 48%;
+  width: auto;
+  margin-top: -23px;
+  padding: 17px;
```
Comment (b0llu) [inline css/home.css]: use rem for above 10px

**Review by b0llu — APPROVED** [review]
Everything else looks great!

File: css/home.css (hunk 4 above)
Comment (ApurvaSawant11) [reply css/home.css]: Changed the units. Thanks

File: css/header.css (hunk 3 above)
Comment (ApurvaSawant11) [reply css/header.css]: Updated it. Thanks

**Pull request merged**